x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/30_semantic_text_inference.yml

```diff
@@ -74,6 +74,93 @@ setup:
   - match: { _source.dense_field.inference.chunks.0.text: "another inference test" }
   - match: { _source.non_inference_field: "non inference test" }
 
+---
+"Calculates text expansion and embedding results for new documents with integer value":
```

Comment:
We're moving away from the term "text expansion", let's use the terms "sparse embedding" (for sparse vectors) and "text embedding" (for dense vectors)